.changes/982.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "internal",
+    "description": "use a generic dockerfiles for when the toolchain and image platfom match."
```

Comment:
if we switch this to be actually used, this should be a change

Reply:
I guess I left this as internal since it doesn't change any existing images at all, since before `aarch64-unknown-linux-gnu` and `x86_64-unknown-linux-gnu` would fail for a `linux/arm64` platform. So, I wasn't sure if this should be changed since it doesn't change anything exposed to a user (although it certainly will add new images), and everything is mostly an internal implementation detail.

Reply:
I had the wrong idea about how this actually works, dismiss this